00_find_bad_subs.py

```diff
@@ -0,0 +1,70 @@
+"""finds bad subjects and saves them as json.
+
+based on either sensitvity index d'
+or binomial test against chance.
+the latter works bc of a clever task design
+To reproduce, you have to delete the bad subs from config.
+"""
+# %%
+# Imports
+import pandas as pd
+from scipy.stats import binom_test
+
+from config import FPATH_DS, SUBJS
+
+# %%
+# settings
+from utils import calculate_dPrime, get_behavioral_data
+
+# %%
+# Filepaths and settings
+rejectionThreshold = 0.6  # dprime after which we exclude participants
+fpath_ds = FPATH_DS
+overwrite = False
+fpath_der = fpath_ds / "derivatives"
+
+# %%
+
+subex = []
+dPrimeEx = []
```

Comment:
again: CamelCase versus snake\_case, see comment above --> I'll not make this comment again for the stuff below :)